apps/client/src/components/QuestionAnswerTable.js

```diff
@@ -41,53 +41,59 @@ export default ({ checker, onGoToQuestion }) => {
       </MainWrapper>
 
       <ComponentWrapper marginBottom={40}>
-        {checker?.stack?.map((question, index) => {
+        {checker?.stack?.map((question, questionIndex) => {
           const isDecisiveForPermits =
-            uniqBy(permitsPerQuestion[index], "name") || [];
-          return (
-            <QuestionWrapper key={question.id}>
-              <Question>{question.text}</Question>
-              {question.options ? (
-                <UserAnswer>{question.answer.replace(/['"]+/g, "")}</UserAnswer>
-              ) : (
-                <UserAnswer>
-                  {
-                    booleanOptions.find(
-                      (option) => option.value === question.answer
-                    ).label
-                  }
-                </UserAnswer>
-              )}
-              {onGoToQuestion && (
-                <Change>
-                  <Button
-                    onClick={() => onGoToQuestion(index)}
-                    variant="textButton"
-                  >
-                    Wijzig
-                  </Button>
-                </Change>
-              )}
-
-              {isDecisiveForPermits.map((permit, index) => (
-                <UserResult key={`${permit} ${index}`}>
-                  <Icon
-                    color={themeColor("secondary", "main")}
-                    size={30}
-                    style={{
-                      flexShrink: 0, // IE11 Fix
-                    }}
-                  >
-                    <Alert />
-                  </Icon>
-                  <UserResultParagraph strong>
-                    Op basis van dit antwoord bent u vergunningplichtig voor{" "}
-                    {permit.name.replace("Conclusie", "").toLowerCase()}
-                  </UserResultParagraph>
-                </UserResult>
-              ))}
-            </QuestionWrapper>
-          );
+            uniqBy(permitsPerQuestion[questionIndex], "name") || [];
+          if (
+            typeof question.answer === "boolean" ||
+            typeof question.answer === "string"
+          ) {
```

Comment:
Why filter only these questions? What are you not interested in? Consider implementing an `else` case. Even if it's only a warning/notise that `type x` is not supported.

Reply:
I don't know why this is. @svenjens why did you add this and can we not remove it? The `sttr-build` is providing us with this information right. For now i've added:
```
} else {
  console.warn(
    `The type "${typeof question.answer}" of "${
      question.answer
    }" is not supported`
  );
}
```

Reply:
Update: removed this check completely